distribution/docs/src/main/resources/content/_configuring/banner-markings-content-extractor.adoc

```diff
@@ -1,5 +1,5 @@
 :title: Configuring Banner Markings Extractor
-:type: configuration
+:type: subSecuring
```

Comment:
Does there need to be a `:link: _banner_markings_extractor` here for this to be properly linked from the hardening checklist bulletpoint [here](https://github.com/codice/alliance/pull/467/files#diff-29a387a036f8d82dc3aa4d6978ee386fR8)?

Reply:
yes. fixed.